webapp/src/toolbox.tsx

```diff
@@ -982,7 +982,7 @@ export class TreeItem extends data.Component<TreeItemProps, {}> {
     renderCore() {
         const { selected, className } = this.props;
         return (
-            <div className={classList(className)} role="treeitem" aria-selected={selected}>
+            <div className={classList(className)} role="treeitem" aria-selected={selected} tabIndex={-1}>
```

Comment:
I don't like the use of `tabIndex={-1}` here, but it's the only way I've managed to work around the focus handler running followed by the click handler.